salt/modules/restartcheck.py

```diff
@@ -314,23 +314,40 @@ def _kernel_versions_nilrt():
             List with possible names of last installed kernel
             as they are probably interpreted in output of `uname -a` command.
     '''
-    kernel_versions = []
-
-    if __grains__.get('os_family') == NILRT_FAMILY_NAME and _is_older_nilrt():
-        # bzImage is copied in the rootfs without any package management or
-        # version info. We also can't depend on kernel headers like
-        # include/generated/uapi/linux/version.h being installed. Even if
-        # we fix this in newer versions of "old NILRT" we still need to be
-        # backwards compatible so it'll just get more complicated.
-        kpath = '/boot/runmode/bzImage'
-        kernel_strings = __salt__['cmd.run']('strings {0}'.format(kpath))
-        re_result = re.search(r'[0-9]+\.[0-9]+\.[0-9]+-rt.*(?=\s\()', kernel_strings)
-        if re_result is not None:
-            kernel_versions.append(re_result.group(0))
+    kver = None
+
+    def _get_kver_from_bin(kbin):
+        '''Get kernel version from a binary image or None if detection fails'''
```

Comment:
Can you split this doc string into 3 lines instead of 1?

Reply:
@rallytime Yes, done.